infrastructure/tooling/src/smoketest/checks/role.js

```diff
@@ -9,19 +9,24 @@ exports.tasks.push({
     'target-roles',
   ],
   run: async () => {
-    let auth = new taskcluster.Auth(taskcluster.fromEnvVars());
-    let randomId = taskcluster.slugid();
-    let roleId = `project:taskcluster:smoketest:${randomId}`;
-    let payload = {
+    const auth = new taskcluster.Auth(taskcluster.fromEnvVars());
+    const randomId = taskcluster.slugid();
+    const roleId = `project:taskcluster:smoketest:${randomId}:*`;
+    const payload = {
       description: 'smoketest test',
-      scopes: ['project:taskcluster:smoketest:*'],
+      scopes: ['project:taskcluster:smoketest:<..>/*'],
     };
-    let expandPayload = {
-      scopes: [`assume:${roleId}`],
+    const expandPayload = {
+      scopes: [`assume:${roleId}abc`],
```

Comment:
This ends up as `assume:project:taskclsuter:smoketest:${randomId}:*abc`, which is why the results below have an extra `*` in them.  Rather than using `${roleId}` here, let's use `assume:project:taskclsuter:smoketest:${randomId}:abc` to avoid that issue.

Reply:
Sure, I will change it.